react-app-drawer-component/my-app/src/App.js

```diff
@@ -0,0 +1,38 @@
+import AppDrawer from './AppDrawer';
+import './App.css';
+import { useState } from 'react';
+
+const items = ['About', 'Get Started', 'Sign In'];
+
+function App() {
+  const [title, setTitle] = useState('');
+  const [view, setView] = useState('main');
+  const overlay = (view === 'main') ?
+    { backgroundColor: 'rgba(255, 255, 255, 0)', pointerEvents: 'none' } :
+    { backgroundColor:'rgba(255, 255, 255, 0.25)', pointerEvents:'all' };
+
+  function toggleMenu() {
+    setView(view === 'main' ? '' : 'main');
+  }
+
+  return (
+    <div>
+      <AppDrawer view={view}
+        items={items}
+        heading="Menu"
+        onTitleChange={(item) => setTitle(item)}
+        onMenuToggle={toggleMenu} />
+      <div style={overlay}
+        className="overlay"
+        onClick={() => view!=='main' && toggleMenu()}>
+      </div>
+      <header className="App-header">
+        {title}
```

Comment:
Congratulations! You're the first one to put the title in App, where it should be. In a real app, you'd display not just the title but the contents of the page for that menu item.